python/graphstorm/model/utils.py

```diff
@@ -157,10 +159,13 @@ def save_sparse_embeds(model_path, embed_layer, local_rank, world_size):
                 embs.append(sparse_emb._tensor[idx])
 
             embs = th.cat(embs, dim=0)
+            # This makedirs may cause folder permission error in the distributed mode.
```

Comment:
The description here is vague. Let's be more precise: 1) In which setting directly calling save_sparse_embeds will get permission issues; 2) who is going to call create_sparse_embeds_path?